### ISSUE TITLE
Soil values "averaging" equipment idea

### ISSUE THREAD

**Issue by Dzi4d3k** [DeckerMMIV/FarmSim_Mod_SoilMod#83]
Hello,

After I saw recently released Soil_Weeder specialization I tough about idea of tool which would be able to average soil values within its working area. As we all know when weeds grow too long in one pleace then N and moisture became low. If we buy then such weedy fields, then it is hard to plan optimal work steps.
Some say that is more real to have such irregular yield on filed, but I personally don't like it. Currently one workaround it to deplete values to 0 and start over, but is even more unreal than idea of having tool which will mix soil and average the values.

Regards.

**Comment by DeckerMMIV** [comment]
It was something I tried a long time ago, experimenting with allowing a cultivator to "average" the soil-properties as it was being dragged through the field.

Unfortunately the experiment failed, because the soil-properties became either "maximum" or "zero", no matter what. - The reason for this, was that when a tool is being moved across a field, the tool's work-area in update-frame-2 will cover ~95% of the tool's work-area from update-frame-1.

So basically the 'area' that was "averaged" in frame-1, then in frame-2 the same 'area' (mostly) was again used to calculate an "average" value, so this multiplied itself over and over again, causing the final value to be either "maximum" or "zero".
